### PR TITLE
Add alternative options for downloading Java agent

### PR DESCRIPTION
**Pull request by bm1549**
### What does this PR do? What is the motivation?

Adds additional options for downloading the Java tracer, using `wget`, `curl`, or `ADD`

### Merge instructions

- [x] Please merge after reviewing
- [ ] Pending review from eng/support team member

### Additional notes

My editor removed extraneous whitespace when I did a find-and-replace to add single quotes around the URL. If this is undesired, I can revert this change

### CODE REVIEW

File: content/en/tracing/guide/inferred-service-opt-in.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ For example, if you're using environment variables and you need to rename the pe
 DD_TRACE_PEER_SERVICE_MAPPING=10.0.32.3:my-service
 ```
 
-[1]: https://dtdg.co/latest-java-tracer
+[1]: 'https://dtdg.co/latest-java-tracer'
```
Comment (alai97) [inline content/en/tracing/guide/inferred-service-opt-in.md:101]:
```suggestion
[1]: https://dtdg.co/latest-java-tracer
```

File: content/en/tracing/trace_collection/dd_libraries/java.md [hunk 2]
```diff
@@ -47,9 +47,23 @@ After the Agent is installed, to begin tracing your applications:
 
 1. Download `dd-java-agent.jar` that contains the latest tracer class files, to a folder that is accessible by your Datadog user:
 
+{{< tabs >}}
+{{% tab "curl" %}}
```
Comment (alai97) [inline content/en/tracing/trace_collection/dd_libraries/java.md:51]:
```suggestion
{{% tab "cURL" %}}
```

File: content/en/dynamic_instrumentation/enabling/java.md [hunk 3]
```diff
@@ -26,10 +26,24 @@ The Datadog Dynamic Instrumentation library is supported in JDK version 8 and ab
 2. If you don't already have APM enabled, in your Agent configuration, set the `DD_APM_ENABLED` environment variable to `true` and listening to the port `8126/TCP`.
 
 3. Download `dd-java-agent.jar`:
-
+ {{< tabs >}}
+ {{% tab "curl" %}}
```
Comment (alai97) [inline content/en/dynamic_instrumentation/enabling/java.md:30]:
```suggestion
 {{% tab "cURL" %}}
```

File: content/en/dynamic_instrumentation/enabling/java.md [hunk 4]
```diff
@@ -26,10 +26,24 @@ The Datadog Dynamic Instrumentation library is supported in JDK version 8 and ab
...
    ```shell
    wget -O dd-java-agent.jar 'https://dtdg.co/latest-java-tracer'
    ```
+{{% /tab %}}
+{{% tab "wget" %}}
```
Comment (alai97) [inline content/en/dynamic_instrumentation/enabling/java.md:35]:
```suggestion
{{% tab "Wget" %}}
```

File: content/en/profiler/enabling/java.md [hunk 5]
```diff
@@ -76,9 +76,23 @@ To begin profiling applications:
 
 2. Download `dd-java-agent.jar`, which contains the Java Agent class files:
 
-    ```shell
-    wget -O dd-java-agent.jar 'https://dtdg.co/latest-java-tracer'
-    ```
+{{< tabs >}}
+{{% tab "curl" %}}
```
Comment (alai97) [inline content/en/profiler/enabling/java.md:80]:
```suggestion
{{% tab "cURL" %}}
```

File: content/en/profiler/enabling/java.md [hunk 6]
```diff
@@ -76,9 +76,23 @@ To begin profiling applications:
...
+   ```shell
+   wget -O dd-java-agent.jar 'https://dtdg.co/latest-java-tracer'
+   ```
+{{% /tab %}}
+{{% tab "wget" %}}
```
Comment (alai97) [inline content/en/profiler/enabling/java.md:85]:
```suggestion
{{% tab "Wget" %}}
```

File: content/en/security/application_security/enabling/java.md [hunk 7]
```diff
@@ -33,9 +33,23 @@ You can monitor application security for Java apps running in Docker, Kubernetes
 ### Get started
 
 1. **Update your [Datadog Java library][1]** to at least version 0.94.0 (at least version 1.1.4 for Application Vulnerability Management vulnerability detection features):
+   {{< tabs >}}
+   {{% tab "curl" %}}
```
Comment (alai97) [inline content/en/security/application_security/enabling/java.md:37]:
```suggestion
   {{% tab "cURL" %}}
```

File: content/en/security/application_security/enabling/java.md [hunk 8]
```diff
@@ -33,9 +33,23 @@ You can monitor application security for Java apps running in Docker, Kubernetes
...
+   ```shell
+   wget -O dd-java-agent.jar 'https://dtdg.co/latest-java-tracer'
+   ```
+{{% /tab %}}
+{{% tab "wget" %}}
```
Comment (alai97) [inline content/en/security/application_security/enabling/java.md:42]:
```suggestion
{{% tab "Wget" %}}
```

File: content/en/tracing/trace_collection/dd_libraries/java.md [hunk 9]
```diff
@@ -47,9 +47,23 @@ After the Agent is installed, to begin tracing your applications:
...
+   ```shell
+   wget -O dd-java-agent.jar 'https://dtdg.co/latest-java-tracer'
+   ```
+{{% /tab %}}
+{{% tab "wget" %}}
```
Comment (alai97) [inline content/en/tracing/trace_collection/dd_libraries/java.md:56]:
```suggestion
{{% tab "Wget" %}}
```

**Review by alai97 — APPROVED** [review]
Looks good for docs!

**Review by wantsui — APPROVED** [review]
LGTM, this was very thorough!

**Review by vidurkhannadatadog — APPROVED** [review]

**Pull request merged**